PresAdmin/src/org/icpc/tools/presentation/admin/internal/ClientsControl.java

```diff
@@ -83,6 +84,7 @@ public class ClientsControl extends Canvas {
 	protected int yOrigin = 0;
 	private boolean waitingForRedraw;
 	protected List<String> filter = new ArrayList<>();
+	protected boolean forceDark;
```

Comment:
Could you make this something like a `Boolean forceColorMode`? We don't need to support overriding to light mode now, but at least it would be generic and allow for it later.

Reply:
added light mode too

Reply:
thanks!